libpod/state.go

```diff
@@ -39,17 +39,10 @@ type State interface { //nolint:interfacebloat
 	// the program.
 	ValidateDBConfig(runtime *Runtime) error
 
-	// SetNamespace() sets the namespace for the store, and will determine
-	// what containers are retrieved with container and pod retrieval calls.
-	// A namespace of "", the empty string, acts as no namespace, and
-	// containers and pods in all namespaces will be returned.
-	SetNamespace(ns string) error
```

Comment:
I know this has been discussed but what if that breaks "important user"?

Reply:
This has been fundamentally broken since Podman 0.12.1, so it didn't even last until 1.0 before breaking.  I will be legitimately amazed if anyone is actually using in in the wild.